gym/spaces/sequence.py

```diff
@@ -38,6 +38,11 @@ def __init__(
             None, None, seed  # type: ignore
         )  # None for shape and dtype, since it'll require special handling
 
+    def seed(self, seed: Optional[int] = None) -> list:
+        """Seed the PRNG of this space and the feature space."""
+        super().seed(seed)
```

Comment:
Normally the seeding for composite spaces includes the super seed list
https://github.com/openai/gym/blob/8b744130bc25d0f4fe2eac8bce8e47527a8eee2c/gym/spaces/dict.py#L103

Reply:
Are you saying that I should change the typehint to `seed: Any = None`? I can see how that makes sense.

Reply:
I think this is what I mean
```python
seeds = super().seed(seed)
seeds += self.feature_space.seed(seed)
return seeds
```

Reply:
Ohhh, of course. I'm adding that :)